File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>



int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");

    } 
    while (n < 1 || n > 8);
    for (int i = n; i > 0; i--)
    {
        for (int j = 0; j < n; j++)
        {
            int c = i - 1;
            if (j < c)
            {
                printf(" ");
            }    
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int j = 0; j < n; j++)
        {
            int c = (n - (i - 1));
            if (j < c)
            {
                printf("#");
            }
            
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int
n = 0
while(n <= 0 or n > 8):
    n = get_int("Height: ")
    
    
for i in range(n):
    print(" " * (n - (i +1)), end = "")
    print("#" * (i+1), end ="")
    print("  ", end = "")
    print("#" * (i+1))
        
#    print("#", end="")
#print("")
